cfgov/v1/jinja2/v1/_layouts/base.html

```diff
@@ -99,51 +99,10 @@
         docElement.className = docElement.className.replace( /(^|\s)no-js(\s|$)/, '$1$2' );
     </script>
 
-    {# TODO: jQuery CDN is provided in the head to satisfy
-             Google Tag Manager (GTM) requirements.
-             Ideally GTM would handle its own dependency management
-             and initializing jQuery could be moved to the footer. #}
-    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
-    <script>if (!window.jQuery) { document.write('<script src="{{ static('js/jquery.min.js') }}"><\/script>'); }
-    </script>
-
 </head>
 
 <body>
 
-<!--
-    =========
-    Analytics
-    =========
--->
-    <!-- CLASSIC ANALYTICS -->
-    {# TODO: Remove Google Analytics code snippet once GTM is stable.
-             We are double tagging with classic analytics and tag manager
-             so that we can test any differences between the two.
-             See: https://github.com/cfpb/cfgov-refresh/issues/496 #}
-    <script>
-        var _gaq=[['_setAccount','UA-20466645-3'],['_setDomainName', '.consumerfinance.gov'],['_trackPageview']];
-        (function(d,t){var g=d.createElement(t),s=d.getElementsByTagName(t)[0];
-        g.src=('https:'==location.protocol?'//ssl':'//www')+'.google-analytics.com/ga.js';
-        s.parentNode.insertBefore(g,s)}(document,'script'));
-    </script>
-
-    <!-- GOOGLE TAG MANAGER -->
```

Comment:
Sadly GTM still needs to be in the head --> See https://developers.google.com/tag-manager/quickstart

> Paste this snippet into your website template page so that it appears immediately after the opening `<body>` tag.

I believe some analytics won't report correctly if their script isn't loaded before the corresponding elements are loaded.